Dockerfile.ci

```diff
@@ -3,5 +3,7 @@ FROM quay.io/coreos-assembler/fcos:testing-devel
 COPY . /go/src/github.com/openshift/okd-machine-os
 WORKDIR /go/src/github.com/openshift/okd-machine-os
 RUN source /etc/os-release \
-    && sed "s;FEDORA_COREOS_VERSION;${OSTREE_VERSION%.*};g" Dockerfile.template > /tmp/Dockerfile \
+    && sed "s;\${FEDORA_COREOS_VERSION};${OSTREE_VERSION%.*}+okd;g" Dockerfile.template > /tmp/Dockerfile.generated \
```

Comment:
Not sure if we want both of these changes - I'd leave FCOS version as is and request adding more display names in `oc` to save kernel / crio versions

Reply:
Hm I think we want to differentiate the plain FCOS version from our OKD build. maybe `+okd.$(date +"%Y%m%d%H%M")` would be even better?

Reply:
Removed the commit for now